rgw/v2/tests/s3cmd/reusable.py

```diff
@@ -439,12 +442,15 @@ def lc_validation_at_archive_zone(bucket_name, config):
     log.info(f"total noncurrent objects are {objs_ncurr}")
     objs_diff = objs_total - objs_ncurr
     if config.test_ops.get("test_lc_expiration"):
-        validation_time = int(config.test_ops["days"] * 30)
+        validation_time = int(config.test_ops["days"] * 40)
```

Comment:
any reason here we changed to 40.
debug_interval we are still using 30 ryt?

Reply:
It was failing with 30 seconds interval time, due to many objects being processed